PowerEditor/installer/nativeLang/brazilian_portuguese.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Brazilian Portuguese by Hélio de Souza and Luxy, updated: Jul 2021, for Notepad++ 8.1.2; last update by Marcello, 02 May 2025, for Notepad++ 8.8.1
+<!-- Brazilian Portuguese by Hélio de Souza and Luxy, updated: Jul 2021, for Notepad++ 8.1.2; last update by Marcello, 05 Jun 2025, for Notepad++ 8.8.2
```

Comment:
Please don't guess the version number without the official announcement.
Use the current version number instead.